### DISCUSSION TITLE
Hana JDBC Using Windows Auth

### DISCUSSION THREAD

**Discussion by Ajordan11725** [dbeaver/dbeaver#14310] [Q&A] [answered]
#### System information: 
- Windows 10 enterprise
- DBeaver version 6.3.2
- Java version 1.8

#### Connection specification:
- HANA
- ngdbc.jar
- No tunnels or proxies 

#### Describe the problem you're observing:
Can not connect to Hana with Windows AD permissions

**Comment by uslss** [comment]
could you add log?

**Comment by Ajordan11725** [comment]
> could you add log?
I dont have a log it just says [10]: authentication failed.
I dont see a place where I can select the ability to use windows AD for authentication. I only see server side username and password areas.

**Comment by kai-morich** [comment, marked as answer]
by default HANA supports user/password authentication, For AD integration please follow instructions on configuring LDAP provider, like https://blogs.sap.com/2019/05/07/ldap-based-authentication-for-sap-hana-2.0/
